include/roco2/kernels/high_low.hpp

```diff
@@ -20,6 +23,18 @@ namespace kernels
         {
         }
 
+        roco2::chrono::time_point::duration get_high_time() const {
```

Comment:
we are following STL-alike coding conventions in this project, so I'd prefer getter without the `get_` prefix, like `tag()`.

Reply:
implemented in 3e199f6